sample_apps/run_example.sh

```diff
@@ -152,13 +165,22 @@ function usage_OE() {
 # ---- SEV-SNP Enclave app-specific help/usage output
 function usage_SEV() {
     echo " "
-    echo "For simple_app_under_sev, you can alternatively use this script "
+    echo "For simple_app_under_sev, you use this script "
     echo "to generate the policy supplying the measurement in the policy JSON file:"
-    echo "Pre-populate simple_app_under_sev/platform_data/ with these required files:"
-    echo "  ${Platform_data_files_SEV}"
     echo " "
-    echo "  To setup the example program        : ./${Me} simple_app_under_sev setup_with_auto_policy_generation_for_SEV"
-    echo "  To run and test the example program : ./${Me} simple_app_under_sev run_test"
+    echo "-- (Default) In a SEV-SNP simulated platform, using the Automated Policy Generator:"
+    echo "   - Build and install the SEV-SNP Simulator utility"
+    echo "  To setup the example program        :      ./${Me} simple_app_under_sev setup"
+    echo "  To run and test the example program : sudo ./${Me} simple_app_under_sev run_test"
+    echo " "
+    echo "-- On SEV-SNP enabled hardware platform:"
+    echo "   Pre-populate simple_app_under_sev/platform_data/ with these required files:"
+    echo "   ${Platform_data_files_SEV}"
+    echo " "
+    echo "  To setup the example program        :      ./${Me} simple_app_under_sev setup_with_auto_policy_generation_for_SEV"
+    echo "  To run and test the example program : sudo ./${Me} simple_app_under_sev run_test"
+    echo " "
+    echo "  To cleanup after a run              : sudo ./cleanup.sh"
```

Comment:
Rearranged help messages. Run `run_example.sh --help simple_app_under_sev` to see how these messages look like.